asciidoctorj-core/src/test/java/org/asciidoctor/WhenAsciidoctorLogsToConsole.java

```diff
@@ -135,6 +136,9 @@ public void log(LogRecord logRecord) {
         };
         asciidoctor.registerLogHandler(logHandler);
 
+        // Asciidoctor currently only logs invalid refs if this global var is set
+        JRubyRuntimeContext.get().evalScriptlet("$VERBOSE=true");
```

Comment:
Doesn't this need to be unset again after the test is run? Or do tests not share the JRuby runtime context?

Reply:
It should be fine, hopefully. 
Every Asciidoctor instance gets its own JRuby instance and should have it's own global state if I am not wrong.
But definitely sth to retest.